services/traction/config.py

```diff
@@ -0,0 +1,28 @@
+import os
+
+
+class Config:
```

Comment:
fastapi has a 'settings' module which has more support for loading from files and stuff.. might want to look into that too.